js/everything.js

```diff
@@ -368,12 +380,11 @@ function removeSearchQueryFromURL(url) {
     }
     return url;
 }
-
 function makeListenURL(videoID) {
```

Comment:
This function should be updated to a take an optional second parameter for position/time, etc.